presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2576,6 +2576,28 @@ public void testCreateBucketedTemporaryTable()
         testCreateTemporaryTable(columns, bucketCount, bucketingColumns, inputRows);
     }
 
+    @Test
+    public void testCreateBucketedOnZeroColumnsTemporaryTable()
```

Comment:
nit: ZeroColumn

Curious: Why not just test create a normal bucketed table with zero bucket column?

Reply:
`ZeroColumn`? 

> Curious: Why not just test create a normal bucketed table with zero bucket column?

There are some edge cases when Presto partitions the data on 0 columns (which is effectively a "gather" partitioning). We should look more closely on those edge cases in Presto at some point. But for now it seemed that this fix is acceptable.